apps/judicial-system/web/src/routes/Prosecutor/RestrictionRequest/StepOne/StepOne.tsx

```diff
@@ -48,10 +52,12 @@ export const StepOne: React.FC = () => {
           nationalId: theCase.defendants[0].nationalId,
           noNationalId: theCase.defendants[0].noNationalId,
           citizenship: theCase.defendants[0].citizenship,
+        }).catch(() => {
```

Comment:
Consider removing this catch and handle the error in the useDefendant hook in a similar way as is done in the useCase hook.